pkg/config/vspheresetupuser.go

```diff
@@ -0,0 +1,41 @@
+package config
```

Comment:
Why does this live separately to the `setupuser` code?

Reply:
I've been leaning into `config` a lot lately trying to extract globally accessed environment variables into config objects. None of this is deriving from the environment though, so yeah I'm happy to put it in `setupuser`.

Reply:
I would start by keeping related types close together. These types seem very specific to setting up a user so I would keep them there. 

A useful trick: consider the touch points for changing this functionality. If you _have_ to touch multiple packages to alter the business logic (wiring up doesn't count as business logic) there might be a smell.